src/StyledTest.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import styled from 'styled-components'
+
+export default function StyledTest () {
+
+    const Title = styled.h1`
```

Comment:
Styling should not be inside components functions. Also please fix formatting here.

Reply:
this component was removed as unnecessary